setup.py

```diff
@@ -58,6 +58,6 @@
     package_dir={'':'src'},
     package_data={'': ['*.xsd', '*.xml']},
     test_suite='metaswitch.crest.test',
-    install_requires=["pyzmq", "py-bcrypt", "cyclone==1.0", "cql", "lxml", "msgpack-python", "pure-sasl"],
+    install_requires=["pyzmq", "py-bcrypt", "cyclone==1.0", "cql", "lxml", "msgpack-python", "pure-sasl", "prctl"],
```

Comment:
I'd prefer https://pypi.python.org/pypi/python-prctl/1.6.1 or https://pypi.python.org/pypi/setproctitle/ - they link to Github repos that are explicit about being BSD, whereas https://pypi.python.org/pypi/prctl,